src/games/strategy/triplea/ai/proAI/ProData.java

```diff
@@ -1,43 +1,83 @@
 package games.strategy.triplea.ai.proAI;
 
 import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.PlayerID;
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.data.UnitType;
 import games.strategy.triplea.Properties;
+import games.strategy.triplea.ai.proAI.data.ProPurchaseOption;
+import games.strategy.triplea.ai.proAI.data.ProPurchaseOptionMap;
+import games.strategy.triplea.ai.proAI.util.ProUtils;
+import games.strategy.triplea.attatchments.TerritoryAttachment;
+import games.strategy.triplea.delegate.BattleCalculator;
+import games.strategy.triplea.delegate.Matches;
+import games.strategy.util.IntegerMap;
+import games.strategy.util.Match;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 /**
  * Pro AI data.
  */
 public class ProData {
 
+  private static ProAI proAI;
+  private static GameData data;
+  private static PlayerID player;
+
+  // Default values
   public static double winPercentage = 95;
   public static double minWinPercentage = 75;
   public static boolean areNeutralsPassableByAir = false;
+  public static Territory myCapital = null;
```

Comment:
Please use "Optional<Territory>" to avoid NPE problems. Also you avoid a SRP code smell as well, since an object that has one meaning when it has a value, and another meaning when it does not, fundamentally does not have one responsibility since it has two different meanings that are state and context dependent.